txtorcon/torinfo.py

```diff
@@ -66,6 +66,14 @@ def dump(self, prefix):
             x.dump(prefix)
 
 
+def _get_descriptor(data):
+    try:
+        from stem.descriptor.server_descriptor import RelayDescriptor
+        data = RelayDescriptor(data)
+    except ImportError:
+        pass
+    return data
+
 class ConfigMethod(object):
```

Comment:
Yes, the ImportError will be shown then, when the .use_stem is True and we don't have Stem installed. Also, I am adding the comment for the method.